app.py

```diff
@@ -84,46 +84,46 @@ def upload():
 
 
 	file_uploaded.save(path)  # save to disk
-	print '%s: saved to disk' % uuid
+	print ('%s: saved to disk') % uuid
```

Comment:
No space after between `print` and `(` as per [PEP8](https://www.python.org/dev/peps/pep-0008/).